src/pages/Introduce/Introduce.scss

```
@@ -0,0 +1,32 @@
+@import "../../style/variables.scss";
+@import "../../style/mixin.scss";
+
+.introduce {
+  @include flexCenter;
+  height: 100vh;
+
+  .wrap {
+    border: 1px solid black;
+    width: 20%;
+    height: 55%;
+    border-radius: 15px;
+    padding: 15px;
+
+    img {
+      width: 100%;
+      height: 85%;
+      border-radius: 15px;
+    }
+
+    p {
+      font-weight: bold;
+      margin: 0;
+    }
+    &:hover {
+      color: $green;
+    }
```

```suggestion
    p {
      font-weight: bold;
      margin: 0;
 
    &:hover {
      color: $green;
    }
    }
```
& 연산자가 더 명확하려면 nesting으로 구현해주세요!

---

명성님 감사합니다.